CONTRIBUTE.md

```diff
@@ -45,6 +45,8 @@ The launch script does env setup with default values, pulls public docker images
 
 Now visit [http://frontend.unstract.localhost](http://frontend.unstract.localhost) in your browser.
 
+Modify the `.env` files present in each service to update its runtime behaviour. Make sure to restart the containers after modifying the `.env`.
+
```

Comment:
@chandrasekharan-zipstack Restarting the containers is not same as doing docker compose down and binging them up again. Former just stops and starts the container, the latter removes the container altogether and creates it again.

Also docker compose command can be installed as `docker compose` or `docker-compose`. Hence might be better not to provide actual command?

```suggestion
NOTE: Modify the `.env` files present in each service folder to update its runtime behaviour. Run docker compose up again for the changes to take effect.```